config/config.go

```diff
@@ -69,6 +69,15 @@ type (
 		CallbackProxyAPI string `ini:"callback_proxy_api"`
 	}
 
+	GitlabOauthCfg struct {
+		ClientID         string `ini:"client_id"`
+		ClientSecret     string `ini:"client_secret"`
+		ServerAddress    string `ini:"server_address"`
```

Comment:
What do we think about using `host` as the field name, to echo other configuration fields?